E01-Geometry/Lukas/Geometry/Shapes/Rhombus.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Geometry\Shapes;
+
+use Geometry\ShapeInterface;
+
+class Rhombus implements ShapeInterface
+{
+    public float $side;
+    public float $altitude;
+
+    public function __construct($side, $altitude)
+    {
+        $this->side = $side;
+        $this->altitude = $altitude;
+    }
+
+    public function area(){
+        if($this->side >= $this->altitude)
+        return $this->altitude * $this->side;
+        throw new \InvalidArgumentException('Altitude must be smaller than or equal to sides to form a rhombus.');
```

Comment:
Consider moving this check into constructor as opposed to checking the altitude while it's a bit too late.
Do validation at the moment you receive the parameters.